installation/manual-installation/mongo-versions/README.md

```diff
@@ -0,0 +1,7 @@
+# Supported Mongo Versions
+
+Rocket.Chat currently requires MongoDB from version 2.3 and onwards.
```

Comment:
Now days we use pretty much 3.0 And higher.  What's the latest recommended from meteor?